app/controllers/brapi/v1/studies_controller.rb

```diff
@@ -0,0 +1,178 @@
+          
+          # To check authentication and ownership when ORCID is supported by BrAPI
+          # We currently only retrieve public records. This is already done at query level
+          #if (!row[:user_id] || row[:published] )
+          json_result_array << row
+          #end
+        end
+        
+        # pagination data returned
+        
+        result_count_object = studies_queries.studies_search_query(
+          params['studyType'], params['studyNames'], params['studyLocations'], params['programNames'],
+          params['germplasmDbIds'], params['observationVariableDbIds'], params['active'],
+          params['sortBy'], params['sortOrder'], page, page_size, true)
+        
+        total_count = result_count_object.values.first[0].to_i
+        total_pages = (total_count/page_size.to_f).ceil
+        
+        json_response = { 
+          metadata: json_metadata(page_size, page, total_count, total_pages),
+          result: json_result_array
+        }
+       
+        render json: json_response, except: ["id", "user_id", "created_at", "updated_at", "total_entries_count"]
+      end
+    
+    end
+  end
+
+
+  def shown
+    # accepted params: id
+
+    if !params['id'].present? 
+      raise ActionController::ParameterMissing.new('Not Found')
+    else 
+      studies_queries = Brapi::V1::StudiesQueries.instance
+          
+      result_object = studies_queries.studies_get_query(params['id'])
+      
+      records = result_object.values
+     
+      if records.nil? || records.size == 0
+        render json: { reason: 'Resource not found' }, status: :not_found  # 404
+      else
+        json_result_array = []
+        
+        # any programmatic data manipulation can be done here
+        result_object.each do |row|
+          row = JSON.parse(row["row_to_json"])
+          
+          location_hash = row.extract!("locationDbId", "name", "countryCode", "countryName", "latitute", 
+            "longitude", "altitude")
+          location_hash[:additional_info] = row.extract!("terrain", "soil_type")
+          row[:location] = [location_hash]
+          
+          contact1_hash = row.extract!("email")
+          contact2_hash = {email: row["entered_by_whom_email"], type: "data_introducer"}
+          row.delete("entered_by_whom_email")
+          row[:contacts] = [contact1_hash, contact2_hash]
+          
+          additional_info_hash = row.extract!("studyDescription", "dataProvenance", "dataOwnedBy")
+          row[:additionalInfo] = additional_info_hash
+          
+          # To check authentication and ownership when ORCID is supported by BrAPI
+          # We currently only retrieve public records. This is already done at query level
+          #if (!row[:user_id] || row[:published] )
+          json_result_array << row
+          #end
+        end
+        
+        json_response = { 
+          metadata: json_metadata(0, 0, records.size, 0),
+          result: json_result_array
```

Comment:
Uf... this is complicated as BrAPI is not always very consistent as it input/output structure sometimes depends on the specific call.
As you say, in SeachStudies they show the 'data' area:
https://github.com/plantbreeding/API/blob/master/Specification/Studies/SearchStudies.md

But they don't include it in studies/:id  : 
https://github.com/plantbreeding/API/blob/master/Specification/Studies/StudyDetails.md

So... I will try to contact with them in order to know what is the correct way of doing this...